playwright/masonry/load-more.spec.mjs

```diff
@@ -65,7 +65,10 @@ test.describe('Masonry: scrolls', () => {
     // Scroll a few times to triggle multiple scrolls.
     await page.evaluate(() => {
       // eslint-disable-next-line playwright/no-conditional-in-test
-      const documentElement = document.documentElement || {};
+      const documentElement = document.documentElement || {
```

Comment:
nit: consider defining the default value once, at the top of the file — something like `defaultDocumentElement`

Reply:
DONE